metadata/mainnet/chains.json

```diff
@@ -2197,14 +2236,14 @@
         ]
       },
       "arena-games-platform": {
```

Comment:
we should change the name here too, otherwise old logo will be displayed till the next Portal update